unittests/test_schedulers.py

```diff
@@ -397,7 +397,7 @@ def test_submit(make_job, exec_ctx):
         assert [socket.gethostname()] == minimal_job.nodelist
         assert minimal_job.exitcode == 0
         assert minimal_job.state == 'SUCCESS'
-    elif sched_name in ('slurm', 'squeue', 'pbs', 'torque', 'flux'):
+    elif sched_name in ('slurm', 'squeue', 'pbs', 'torque'):
```

Comment:
Add `flux` here only if there is a way to retrieve the nodes that the job has run on.